tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/PartialViews.spec.ts

```diff
@@ -0,0 +1,240 @@
+﻿import {AliasHelper, ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+test.describe('Partial Views tests', () => {
+  
+  test.beforeEach(async ({page, umbracoApi, umbracoUi}) => {
+    await page.goto(umbracoApi.baseUrl + '/umbraco');
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
+  });
+
+  async function gotoPartialViewDetail(page, partialViewFileName) {
+    await page.locator('umb-tree-item').filter({ hasText: 'Partial Views' }).locator('#caret-button').click();
+    await page.locator('umb-tree-item').getByLabel(partialViewFileName).click();
+  }
+  
+
+  test('can create an empty partial view', async ({page, umbracoApi}) => {
+    const partialViewName = 'TestPartialView';
+    const partialViewFileName = partialViewName + ".cshtml";
+    // Act
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewName);
+
+    await page.locator('umb-tree-item', {hasText: 'Partial Views'}).getByLabel('Open actions menu').click({force: true});
+    await page.getByLabel('New empty partial view').click();
+    await page.getByLabel('template name').fill(partialViewName);
+    // TODO: Remove this timeout when frontend validation is implemented
+    await page.waitForTimeout(1000);
+    await page.getByLabel('Save').click();
+
+    // Assert
+    expect(await umbracoApi.partialView.nameExistsAtRoot(partialViewFileName)).toBeTruthy;
+    // TODO: when frontend is ready, uncomment this to verify the new partial view is displayed under the Partial Views section 
+    //await expect(page.locator('umb-tree-item', {hasText: 'Partial Views'}).getByLabel(partialViewFileName)).toBeVisible();
+    // TODO: when frontend is ready, verify the notification displays
+    
+    // Clean
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewFileName);
+  });
+
+  test('can create a partial view from snippet', async ({page, umbracoApi}) => {
+    const partialViewName = 'TestPartialViewFromSnippet';
+    const partialViewFileName = partialViewName + ".cshtml";
+    // Act
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewFileName);
+
+    await page.locator('umb-tree-item', {hasText: 'Partial Views'}).getByLabel('Open actions menu').click({force: true});
+    await page.getByLabel('New partial view from snippet...').click();
+    await page.getByLabel('Breadcrumb').click();
+    await page.getByLabel('template name').fill(partialViewName);
+    // TODO: Remove this timeout when frontend validation is implemented
+    await page.waitForTimeout(1000);
+    await page.getByLabel('Save').click();
+    
+
+    // Assert
+    expect(await umbracoApi.partialView.nameExistsAtRoot(partialViewFileName)).toBeTruthy;
```

Comment:
We should have an expect for checking that the snippet code we got from selecting the breadcrumb is correct